CONTRIBUTING.md

```diff
@@ -33,6 +33,7 @@ $ rubocop -V
 * If your change has a corresponding open GitHub issue, prefix the commit message with `[Fix #github-issue-number]`.
 * Make sure to add tests for it. This is important so I don't break it
   in a future version unintentionally.
+* If you are adding a new cop add an entry for it in `config/default.yml`.  Use `VersionAdded: '<<next>>'`.
```

Comment:
I think it's probably better to explanation rake task `new_cop` instead.

```suggestion
* If you are adding a new cop [use the bundled rake task `new_cop` to generate a cop template][7].
```

If so, please update the entry and add the following link at the end.

```markdown
[7]: https://docs.rubocop.org/rubocop/development.html
```

Reply:
indeed, that's what i should have started with 😅